.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
+
+## How do I contribute?
+
+There are two logical avenues to contribute to the code and content usrse.github.io, including (but not limited to):
+
+ - opening up an issue to ask a question, report a bug, or request a feature
+ - working on a change to the website and opening up a pull request to discuss
+
+This guide will help you to communicate with usrse (if needed) and guide you with
+proper steps to make your contribution.
+
+## Communication
+
+Whether you are opening a pull request or an issue, you might want to communicate with
+others in the community first. How can you do this? You can:
+
+ - have discussion on the USRSE slack (usrse.slack.com) in the `#website` channel
+ - open an issue at https://www.github.com/usrse/usrse.github.io
+ - write an email to contact@us-rse.org (for more private questions)
+
+It could be that you want to talk about a new feature or bug, and then follow up by opening
+an issue at https://www.github.com/usrse/usrse.github.io/issues. It could also be
+that you want to suggest and then work on a new feature, but you want to check first
+that a pull request would be welcomed. Regardless of your need, the above channels
+can offer you a means to do this.
+
+### Teams
+
+We have several teams on the repository that you can reference (@ "at") if you need help!
+
+ - @usrse-admin: should be used for more important discussion that should involve administrators.
+ - @usrse-maintainers: are referenced when a new pull request is open, and are primarily responsible for doing or assigning review.
+
+## Making a Contribution
+
+### Pull Requests
+
+When making a pull request contribution, it is important to properly communicate the
+gist of the contribution. If it is a simple code or editorial fix, simply
+explaining this within the GitHub Pull Request (PR) will suffice. But if this
+is a larger fix or enhancement, it should be first discussed with the project
+leader or developers. See the [communication channels](#communication) for ways
+to do this.
+
+#### Pull Request Process
+
+The repository comes with a [pull request template](PULL_REQUEST_TEMPLATE.md) that provides you with a checklist and
+helpful reminders for these points.
+
+1. All pull requests should be sent to the master branch, unless you are testing an integration, in which case you should cc @usrse-admin in an issue, or ask for help on the website channel of usrse.slack.com.
+2. Follow the existing code style precedent.
+3. Test and preview your PR locally, and tell the reviewers if you were not able to
+4. If necessary, update the README.md and CHANGELOG.md.
```

Comment:
What should we consider necessary?  What's the threshold?  Anything more than a tiny fix?  Or just the substantial changes?

Reply:
Changelogs usually call for significant changes that, if you zoom ahead into the future, someone might want to know about. Examples might be adding an entire new section, re-organizing or restyling, or anything that changes a state to something different. Changes that aren't necessary are small bug fixes, content tweaks, etc. It's really less of a "This is the law!" sort of deal and more of "if your intuition says this is a big change, add to the log." This is very different than maintaining software, so generally I'd say people would err on the side of not adding content. It's really up to the author and reviewers of the PR.

Reply:
(hence why I wrote if necessary - if necessary is really meaning "If you think it's necessary, or if your reviewers look at the changes and think it's necessary).

Reply:
What do you think about adding some text like this (or even this) to contributing.md?  Just so people know?  Or put a bit more at the top of changelog itself?

Reply:
On it!